packages/tables/resources/views/components/columns/column.blade.php

```diff
@@ -42,7 +42,7 @@
         <a
             href="{{ $url ?: $recordUrl }}"
             {!! $shouldOpenUrlInNewTab ? 'target="_blank"' : null !!}
-            class="block"
+            class="inline-block"
```

Comment:
@danharrin, this broke `recordUrl()` since the column is no longer full width and thus only the text part of the column can be clicked instead of the entire row.

Reply:
The button can be made to fill the cell with the following css.
<img width="706" alt="Screenshot 2023-01-25 at 11 09 41" src="https://user-images.githubusercontent.com/533658/214548444-c2c14bc5-a34b-4471-b097-157f2e78b454.png">

Reply:
we already fixed